frontend/src/components/Candidate.jsx

```diff
@@ -0,0 +1,31 @@
+export default function Candidate() {
```

Comment:
'Candidate' doesn't reflect what the component is. Maybe 'NewCandidateForm' ?

Reply:
renamed the component name in NewCandidateForm
and compagny.jsx is now NewConsultantForm